app/assets/stylesheets/style.scss

```diff
@@ -12,15 +13,20 @@ body {
     max-height: 100px;
   }
 }
+
 .scrollable{
-  width:150px;
-  max-width: 150px;
+  width:140px;
```

Comment:
Properties should be ordered color, height, max-height, max-width, overflow-y, width<br>Colon after property should be followed by one space